generated/PYPI_README.md

```diff
@@ -163,11 +163,20 @@ release provided they have access to the appropriate platform and tools.
 
 ## Contributing
 
-Want to help build Apache Airflow? Check out our [contributing documentation](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst).
+Want to help build Apache Airflow? Check out our [contributing documentation](https://github.com/apache/airflow/blob/main/contributing-docs/README.rst).
 
 Official Docker (container) images for Apache Airflow are described in [IMAGES.rst](https://github.com/apache/airflow/blob/main/IMAGES.rst).
 
 
+## Commit Policy
+
+* Commits need a +1 vote from a committer who is not the author
+* Do not merge a PR that regresses linting or does not pass CI tests (unless you have good
+  justification that it a transient error or something that is being fixed in other PR).
+* We do not merge PRs that have unresolved conversation. Note! This is experimental - to be assessed by the
+  end of January 2024 if we want to continue it.
+* When we do AIP voting, both PMC member's and committer's `+1s` are considered a binding vote.
```

Comment:
Is AIP policy for voting correct in this section?

Reply:
Yes. For AIPs committers have binding votes.  It's only for releases that PMC members have binding votes (this is also legal requirement by the ASF, because releasing software is a legal act of the Foundation and it has legal consequences).

Reply:
That's the relevant ASF rule: https://www.apache.org/legal/release-policy.html#release-approval

Reply:
I was rather referring to the fact that this chapter is about merging _PRs_ and I understand that AIP's voting rules - but AIP's are not _merged_.

Reply:
Ah  you are quite right. This one was mixing things. I split it. I renamed this README part to be "Voting policy" and move the "condition of merging" to the "Pull Request" doc of contribution-docs.